weave-js/src/components/Panel2/ChildPanel.tsx

```diff
@@ -565,6 +565,7 @@ export const ChildPanel: React.FC<ChildPanelProps> = props => {
     useElementWidth<HTMLDivElement>();
 
   const controlBar = props.controlBar ?? 'off';
+  const isVarNameEditable = props.editable || controlBar === 'editable';
```

Comment:
does it ever make sense for `editable = False` but `controlBar = 'editable`? I guess more generally, can you add a comment that explains the intended usage of these params? And if all 6 combos make sense?